src/adapters/Celer/CelerReceiverAdapter.sol

```diff
@@ -13,9 +13,7 @@ import "./libraries/Utils.sol";
 
 interface IMessageReceiverApp {
     enum ExecutionStatus {
```

Comment:
do we really need an enum if all we have is a single value? Can we just return a bool in `executeMessage()`?